src/index.js

```diff
@@ -49,7 +54,18 @@ let fightInfo = {
   startingEnemyHp: enemy.hp,
 };
 
+let inventory = [];
+
 // Command that begins fight
+
+const checkInv = () => {
+  let message = '';
+  for (let i = 0; i < inventory.length; i++) {
+    message += inventory[i].mat + ' | ' + inventory[i].quantity + '\n';
+  }
```

Comment:
99% of the time you can use a `for..of` loop. Classic style indexed for loops like this are the way of the past (and much more error prone!).

On another note, it is usually preferred to use the backtick template strings when building strings out of other strings, as it's much more readable.


```suggestion
  for (const item of inventory) {
    message += `${item.mat} | ${item.quantity}\n`;
  }
```